habitat-lab/habitat/core/env.py

```diff
@@ -128,6 +128,28 @@ def __init__(
             }
         )
         self.action_space = self._task.action_space
+
+        # Sharing controller between actions
```

Comment:
This is loading the human controller for all environments? This should definitely go elsewhere. Why not in the action controller?

Reply:
Where is the action controller?

Reply:
Thanks for the feedback! Addressed in latest commit.